doc/concepts/embeddings.rst

```diff
@@ -0,0 +1,43 @@
+.. _embeddings:
+
+Quantum Embedding
+==================
+
+Quantum Embedding is the process of representing classical data as quantum states in a Hilbert space, i.e., *a quantum feature map*. This process is a crucial part of designing quantum algorithms and hence their complexity. For details, see [#]_ and [#]_. PennyLane provides built-in embedding templates that can do the embedding for the user; see :ref: `<pennylane.templates.embeddings>`. 
+
+Lets consider classical input data consisting of M examples, N dimensional each: :math:`\mathcal{D}=\{x^{1}, \ldots, x^{M}\}`. To embed this onto an n-bit quantum system (n qubits *or* n qumodes for discrete and continuous variable quantum computing, respectively), we can use various embedding techniques some of whcih are explained breifly below. 
```

Comment:
```suggestion
Let's consider classical input data consisting of :math:`M` examples, with :math:`N` dimensions each: :math:`\mathcal{D}=\{x^{1}, \ldots, x^{M}\}`. To embed this data into quantum system (n qubits *or* n qumodes for discrete and continuous variable quantum computing, respectively), we can use various embedding techniques, some of which are explained briefly below. 
```

Reply:
Would be useful to have a higher-level description of embeddings before jumping to specific ones. i.e., an embedding takes a classical datapoint x and translates this into a set of gate parameters in a quantum circuit. This ends up creating a state |\psi_x> on the quantum system.